src/monocle_apptrace/wrap_common.py

```diff
@@ -148,7 +151,14 @@ def update_llm_endpoint(curr_span: Span, instance):
         if 'temperature' in instance.__dict__:
             temp_val = instance.__dict__.get("temperature")
             curr_span.set_attribute("temperature", temp_val)
-        # handling for model name
+        if 'document_store' in instance.__dict__:
```

Comment:
can we group the code so that it would be easy to manage ? Triton code seems to be seperate from llama index and langchain